test/client/HomeControllerSpec.js

```diff
@@ -0,0 +1,33 @@
+var expect = chai.expect;
+
+describe('HomeController', function() {
+  var $scope, $rootScope;
+
+  beforeEach(module('Home'));
+
+  beforeEach(inject(function($injector) {
+    $rootScope = $injector.get('$rootScope');
+
+    $scope = $rootScope.$new();
+
+    var $controller = $injector.get('$controller');
+
+    createController = function() {
+      return $controller('HomeController', {
+        $scope: $scope
+      });
+    };
+
+    createController();
+  }));
+
+  // checking for test is just a test to test the tests
```

Comment:
You really don't need to copy this line over; it's only relevant in the User controller tests.